src/linter/binding/BindingLinter.ts

```diff
@@ -136,7 +143,9 @@ export default class BindingLinter {
 			}
 		} catch (err) {
 			const message = err instanceof Error ? err.message : String(err);
-			this.#context.addLintingMessage(this.#resourcePath, MESSAGE.PARSING_ERROR, {message}, position);
+			if (err instanceof Error) {
```

Comment:
I think this still comes from the other change about improving the position for errors. I'm not sure whether `err` actually always is an instance of Error, but if this is ensured, we can also adjust the line that handles the message.